caffe2/operators/gelu_op.cc

```diff
@@ -0,0 +1,132 @@
+#include "caffe2/operators/gelu_op.h"
+
+#include <algorithm>
+#include <cmath>
+#include <functional>
+#include <numeric>
+#include <string>
+#include <vector>
+
+#include "caffe2/utils/eigen_utils.h"
+#include "caffe2/utils/math.h"
+
+namespace caffe2 {
+
+template <>
+template <typename T>
+bool GeluFunctor<CPUContext>::
+operator()(const int N, const T* X, T* Y, CPUContext* context) const {
+  ConstEigenVectorArrayMap<T> X_arr(X, N);
+  EigenVectorArrayMap<T> Y_arr(Y, N);
+  if (fast_gelu) {
```

Comment:
add more comments? e.g., the equation of fast-gelu and gelu? maybe the link of the paper?

Reply:
Done